validation/src/KrigingGu/compute_validation_values.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Test and validate OpenTURNS Kriging Algorithm with Gu methods. 
+"""
+import openturns as ot
+import sys
+sys.path.insert(0,'.')
+from Classe_Gu import Gu
```

Comment:
Shouldn't this be ``
from Gu import Gu
``?

Reply:
yes, fixed